packages/sdk/src/swap/sdk.ts

```diff
@@ -494,7 +494,7 @@ export class SwapSDK {
         channelOpeningFee: result.channelOpeningFee,
       };
     } else {
-      response = await this.trpc.openSwapDepositChannel.mutate(depositAddressRequest);
+      response = await this.trpc.openSwapDepositChannel.mutate({ ...depositAddressRequest, quote });
```

Comment:
should we save a quote if the broker is defined too?

Reply:
i thought about it, but im not sure. it would be a separate request after the channel was created to collect the data, this will feel a bit like "phoning home" for integrators